### ISSUE TITLE
Interesting issue could be my ISP

### ISSUE THREAD

**Issue by Lordbeatmeater69** [KevinLage/YouTube-Livestream-Botter#180]
SO i used this program for my friends and they got a lot of growth, not the views are not being added and when i go to give them help i get this weird issue with SSL, i go to update the proxy's and the site cant be reached. I use my VPN and then everything works great! But the views are still not added and the watch time doesn't get added if i use a VPN. Not sure how i can go about fixing this :P
![image](https://user-images.githubusercontent.com/74317007/98841043-cd6eec80-2404-11eb-92f3-58467e1a4c97.png)
![image](https://user-images.githubusercontent.com/74317007/98841128-ea0b2480-2404-11eb-881e-06339096300c.png)

**Comment by qveenqueenie** [comment]
I have the same issue. Except i dont have a VPN... Everything worked fine around september but now i get this same issue... I turned off the security settings toy chromr and i can acess the website but o still get the same error code, and idk how to ask for baby steps on how to resolve it or fix it... Lol...

**Comment by Lordbeatmeater69** [comment]
Yeah only way ive seen it kinda work still is with a VPN but OBS doesn't allow streaming with a VPN so its kinda a shot in the dark if it works. ive had it work once sense September, i still try it here and there to see if it works or not. Lets see if Mr Kevin can get a fix in the works :)

**Comment by qveenqueenie** [comment]
> Yeah only way ive seen it kinda work still is with a VPN but OBS doesn't allow streaming with a VPN so its kinda a shot in the dark if it works. ive had it work once sense September, i still try it here and there to see if it works or not. Lets see if Mr Kevin can get a fix in the works :) 

He saod it might be my mcafee but i completely deleted it and he said it might be because of a security thing or my isp

**Comment by Lordbeatmeater69** [comment]
Ive looked into security, i deleted my anti virus just in case it was blocking it. didnt work, changed to a public DNS still didnt work, only thing working is VPN. But still doesn't work correctly.

**Comment by qveenqueenie** [comment]
> Ive looked into security, i deleted my anti virus just in case it was blocking it. didnt work, changed to a public DNS still didnt work, only thing working is VPN. But still doesn't work correctly.

EXACTILY

**Comment by KevinLage** [comment]
Try to install python and run the python script

might help

**Comment by qveenqueenie** [comment]
> Try to install python and run the python script
> 
> might help

Is that something under ur releases, or is it just something i download from searching for it?

**Comment by KevinLage** [comment]
you can install python from pythom.org and download the .py from the repo

**Comment by qveenqueenie** [comment]
> you can install python from pythom.org and download the .py from the repo

Okie. Hehe thankyou for being patient with me.. 🙈

**Comment by qveenqueenie** [comment]
Whats the .py variant of the program and where do i go to download it? Im still nee to github so i might need simple instructions... I have the program from python.org downloaded and etc. I just dont know what to do or where to go next... Help?
